.github/workflows/tests.yml

```diff
@@ -18,3 +18,6 @@ jobs:
         ghcr.io/product-os/flowzone
       balena_slugs: |
         product_os/flowzone
+      cargo_targets: |
+        x86_64-unknown-linux-gnu,
+        aarch64-unknown-linux-gnu
```

Comment:
We probably also want armv7 since most of our devices use that and I think we'd want our rust builds to work on most devices we support?

Reply:
Sure, I can add it, although this just tests that flowzone can build for multiple architectures

Reply:
Sure, I can add it